api/controllers/v0.2/createPatron.js

```diff
@@ -55,74 +55,154 @@ function renderResponse(req, res, status, message) {
 }
 
 /**
- * createPatron(req, res)
- * The callback for the route "/patrons".
- * It will fire a POST request to the ILS API for creating a new patron.
+ * streamPatron(streamPatron, modeledResponse)
```

Comment:
I assume the parameters will be like `(req, res, streamPatron, modeledResponse)`? If this is the case, line 58 should be `streamPatron(req, res, streamPatron, modeledResponse)`